packages/terra-form-select/src/shared/_SearchResults.module.scss

```diff
@@ -0,0 +1,7 @@
+:local {
+  .search-results {
+    color: #8d8d8d;
+    padding: 6px;
+    font-weight: 500;
+  }
+}
```

Comment:
Do we need the style changes ..?? As UX has not provided any style changes.

Reply:
Removed styles and updated